### PR TITLE
[main] Add Konflux configurations

### PR DESCRIPTION
**Pull request by serverless-qe**
Add Konflux components and pipelines

### CODE REVIEW

File: .konflux/applications/serverless-operator-135/components/imagerepositories/serverless-source-image-135.yaml
Comment (pierDipi) [inline .konflux/applications/serverless-operator-135/components/imagerepositories/serverless-source-image-135.yaml:1]: This is not needed

Comment (creydr) [reply .konflux/applications/serverless-operator-135/components/imagerepositories/serverless-source-image-135.yaml:1]: https://github.com/openshift-knative/hack/pull/343

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**